internal/cli/sketch/new.go

```diff
@@ -38,7 +39,14 @@ func initNewCommand() *cobra.Command {
 		Long:    tr("Create a new Sketch"),
 		Example: "  " + os.Args[0] + " sketch new MultiBlinker",
 		Args:    cobra.ExactArgs(1),
-		Run:     func(cmd *cobra.Command, args []string) { runNewCommand(args, overwrite) },
+		Run:     func(cmd *cobra.Command, args []string) {
+			re := regexp.MustCompile("^[a-zA-Z].")
+			if !re.MatchString(args[0]) {
```

Comment:
You are testing the entire path here. The specification must be applied only to the sketch folder name.

This is the reason the unit tests are failing. Even though they have a valid sketch folder name, the path starts with `/`, which is not allowed by your regex.

Reply:
Thanks for the quick reply, I'm working on those changes now.